definitions/m5-release-pipeline/m5-release-pipeline.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     - name: clone-config-file
       taskRef:
         name: git-clone
-        bundle: quay.io/hacbs-release/git-clone:main
+        bundle: quay.io/redhat-appstudio/appstudio-tasks:583d33c8ddf0de2ea8e1a73d94a1ca4a6e6ed380-1
```

Comment:
That looks correct.
FYI here is a quick one-liner to inspect that bundle, and it does indeed contain the git-clone.
```
tkn bundle list --no-cache --output "json" quay.io/redhat-appstudio/appstudio-tasks:583d33c8ddf0de2ea8e1a73d94a1ca4a6e6ed380-1 | jq -r '{kind: .kind, name: .metadata.name, steps: {name: .spec.steps[].name, image: .spec.steps[].image, params: [.spec.params[]] } } '
...
{
  "kind": "Task",
  "name": "git-clone",
  "steps": {
    "name": "clone",
    "image": "$(params.gitInitImage)",
    "params": [
      {
        "name": "url",
        "type": "string",
        "description": "Repository URL to clone from."
      },
      {
        "name": "revision",
        "type": "string",
        "description": "Revision to checkout. (branch, tag, sha, ref, etc...)",
        "default": ""
      },
      {
        "name": "refspec",
        "type": "string",
        "description": "Refspec to fetch before checking out revision.",
        "default": ""
      },
      {
        "name": "submodules",
        "type": "string",
        "description": "Initialize and fetch git submodules.",
        "default": "true"
      },
      {
        "name": "depth",
        "type": "string",
        "description": "Perform a shallow clone, fetching only the most recent N commits.",
        "default": "1"
      },
      {
        "name": "sslVerify",
        "type": "string",
        "description": "Set the `http.sslVerify` global git config. Setting this to `false` is not advised unless you are sure that you trust your git remote.",
        "default": "true"
      },
      {
        "name": "subdirectory",
        "type": "string",
        "description": "Subdirectory inside the `output` Workspace to clone the repo into.",
        "default": ""
      },
      {
        "name": "sparseCheckoutDirectories",
        "type": "string",
        "description": "Define the directory patterns to match or exclude when performing a sparse checkout.",
        "default": ""
      },
      {
        "name": "deleteExisting",
        "type": "string",
        "description": "Clean out the contents of the destination directory if it already exists before cloning.",
        "default": "true"
      },
      {
        "name": "httpProxy",
        "type": "string",
        "description": "HTTP proxy server for non-SSL requests.",
        "default": ""
      },
      {
        "name": "httpsProxy",
        "type": "string",
        "description": "HTTPS proxy server for SSL requests.",
        "default": ""
      },
      {
        "name": "noProxy",
        "type": "string",
        "description": "Opt out of proxying HTTP/HTTPS requests.",
        "default": ""
      },
      {
        "name": "verbose",
        "type": "string",
        "description": "Log the commands that are executed during `git-clone`'s operation.",
        "default": "true"
      },
      {
        "name": "gitInitImage",
        "type": "string",
        "description": "The image providing the git-init binary that this Task runs.",
        "default": "registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:af7dd5b3b1598a980f17d5f5d3d8a4b11ab4f5184677f7f17ad302baa36bd3c1"
      },
      {
        "name": "userHome",
        "type": "string",
        "description": "Absolute path to the user's home directory. Set this explicitly if you are running the image as a non-root user or have overridden\nthe gitInitImage param with an image containing custom user configuration.\n",
        "default": "/tekton/home"
      }
    ]
  }
}
...
```

Looks like by default they use `registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:af7dd5b3b1598a980f17d5f5d3d8a4b11ab4f5184677f7f17ad302baa36bd3c1`